extensions/ohif-i18n/src/locales/en/Buttons.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Themes": "Themes",
+  "Previous": "$t(Common:Previous)",
+  "Next": "$t(Common:Next)",
+  "Play": "$t(Common:Play)",
+  "Stop": "$t(Common:Stop)",
+  "Layout": "$t(Common:Layout)",
+  "More": "$t(Common:More)",
+  "Crosshairs": "Crosshairs",
+  "Magnify": "Magnify",
+  "ROI Window": "ROI Window",
+  "Probe": "Probe",
+  "Ellipse": "Ellipse",
+  "Rectangle": "Rectangle",
+  "Invert": "Invert",
+  "Rotate Right": "Rotate Right",
+  "Flip H": "Flip H",
+  "Flip V": "Flip V",
+  "Clear": "Clear",
+  "Brush": "Brush",
+  "Coronal": "Coronal",
+  "Stack Scroll": "Stack Scroll",
+  "Measurements": "Measurements",
+  "Zoom": "Zoom",
+  "Levels": "Levels",
+  "Pan": "Pan",
+  "Length": "Length",
+  "Angle": "Angle",
+  "Bidirectional": "Bidirectional",
+  "Freehand": "Freehand",
+  "EllipticalRoi": "EllipticalRoi",
+  "CircleRoi": "CircleRoi",
+  "RectangleRoi": "RectangleRoi",
+  "Reset": "$t(Common:Reset)",
+  "Reset 2": "$t(Common:Reset) 2",
+  "CINE": "CINE",
+  "Acquired": "Acquired",
+  "Sagittal": "Sagittal",
+  "Axial": "Axial",
+  "Manual": "Manual",
+  "Manual 2": "Manual 2",
```

Comment:
What are Manual 2 and Reset 2?

Reply:
They are in react-viewerbase https://react.ohif.org/components/toolbar-section - Click on The more menu and you will see them. We could change the example there to something more "translatable"?